doc/hash/hash-design.md

```diff
@@ -427,10 +431,10 @@ hash-algorithm  = ""
 redis-cli -n 4 HGETALL 'SWITCH_HASH|GLOBAL'
 1) "ecmp_hash@"
 2) "DST_MAC,SRC_MAC,ETHERTYPE,IP_PROTOCOL,DST_IP,SRC_IP,L4_DST_PORT,L4_SRC_PORT, \
-INNER_DST_MAC,INNER_SRC_MAC,INNER_ETHERTYPE,INNER_IP_PROTOCOL,INNER_DST_IP,INNER_SRC_IP,INNER_L4_DST_PORT,INNER_L4_SRC_PORT"
+INNER_DST_MAC,INNER_SRC_MAC,INNER_ETHERTYPE,INNER_IP_PROTOCOL,INNER_DST_IP,INNER_SRC_IP,INNER_L4_DST_PORT,INNER_L4_SRC_PORT,IPV6_FLOW_LABEL"
```

Comment:
Just wondering if this is going to be the default hash or optional?

Reply:
@prsunny all hash field configuration is optional. Here we do not change vendor SAI defaults but only add a new configuration capability. If you want to change SAI vendor defaults - please let us know